doc/generic/pgf/text-en/pgfmanual-en-tikz-matrices.tex

```diff
@@ -475,6 +475,53 @@ \subsubsection{Cell Styles and Options}
 \end{tikzpicture}
 \end{codeexample}
 
+In some cases, it is desirable to include some automation in each column/row
+separately. A typical example is to apply stripe-pattern to almost all columns
+and treat only a few of them differently. Then, nesting these keys can open up
+a lot of possibilities; in the following example a "feature comparison" table
```

Comment:
```suggestion
a lot of possibilities; in the following example a ``feature comparison'' table
```

Reply:
Eyh...these stuff... I had happily forgotten TeX :-) 

Are you deploying the build artifacts somewhere I can see the end result quickly?